src/utilities/Validation.js

```diff
@@ -0,0 +1,80 @@
+const users = require("../data/users.json");
+
+class Validation {
+  static validateSignup = (userBody) => {
```

Comment:
Please explore some open-source options to do basic data type checks, required field checks, and even the complicated validations.